Content.Server/_Harmony/GameTicking/Rules/Components/ConspiratorRuleComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared.Random;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server._Harmony.GameTicking.Rules.Components;
+
+/// <summary>
+/// Game rule for conspirators. Handles their shared objective.
+/// </summary>
+[RegisterComponent, Access(typeof(ConspiratorRuleSystem))]
+public sealed partial class ConspiratorRuleComponent : Component
+{
+    public EntProtoId? Objective = null;
+    public ProtoId<WeightedRandomPrototype> ObjectiveGroup = "ConspiratorObjectiveGroup";
```

Comment:
These should be datafields so they can be saved and loaded to yaml.
Some comments explaining what each of those do would also be nice
```suggestion
    [DataField]
    public EntProtoId? Objective = null;
    
    [DataField]
    public ProtoId<WeightedRandomPrototype> ObjectiveGroup = "ConspiratorObjectiveGroup";
```